p11/session.go

```diff
@@ -40,6 +40,8 @@ type Session interface {
 	GenerateKeyPair(request GenerateKeyPairRequest) (*KeyPair, error)
 	// GenerateRandom returns random bytes generated by the token.
 	GenerateRandom(length int) ([]byte, error)
+	// Wrap key returns ciphertext bytes from wrapping a key with another key
+	WrapKey(request WrapKeyRequest) ([]byte, error)
```

Comment:
This session interface starts looking like a dumping ground for methods, can you find a better way than adding another method here?